docs/cli_README.md

```diff
@@ -5,21 +5,52 @@
 ### Installation:
 - Select, Download and Install latest binary from [Releases](https://github.com/fabric8-analytics/cli-tools/releases)
 
+#### curl
+
+Example:
+```
+# For Linux 64Bit, Downloading v0.0.1
+$. curl -O -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux_64bit.tar.gz
```

Comment:
I guess it would be good to have this per platform(not only linux, windows and mac as well). Usually people love to copy and paste to install some tools instead of applying some brain power.

Reply:
Ack